projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -792,21 +792,36 @@ fluentd:
 #      some system config
 #    </system>
 
-## These values expose configuration properties for the execution api
-## The execution API helps you collect information about previous
-## data job executions, or manually start a job execution. Stored data job
-## execution information will naturally increase over time. This is why we
-## are providing a way to clean old information about data job executions through
-## the following properties:
-##
-## maximumExecutionsToStore - The maximum amount of executions per data job to keep. If data
-##                            job executions exceed this number, the older executions are deleted.
-##
-## ttlSeconds - Total time to live per data job execution. Data job executions older than ttlSeconds
-##              specified also get deleted.
-##
-dataJobExecutionsCleanupTask:
-  ## maximum number of data job executions to store in the database / default is 100.
-  maximumExecutionsToStore: "100"
-  ## maximum time to live seconds for each execution / default is 14 days.
-  executionsTtlSeconds: "1209600"
+dataJob:
+  executions:
+    ## These values expose configuration properties for the execution api
+    ## The execution API helps you collect information about previous
+    ## data job executions, or manually start a job execution. Stored data job
+    ## execution information will naturally increase over time. This is why we
+    ## are providing a way to clean old information about data job executions through
+    ## the following properties:
+    ##
+    ## maximumExecutionsToStore - The maximum amount of executions per data job to keep. If data
+    ##                            job executions exceed this number, the older executions are deleted.
+    ##
+    ## executionsTtlSeconds - Total time to live per data job execution. Data job executions older than ttlSeconds
+    ##              specified also get deleted.
+    ##
+    cleanupTask:
+      ## maximum number of data job executions to store in the database / default is 100.
+      maximumExecutionsToStore: "100"
+      ## maximum time to live seconds for each execution / default is 14 days.
+      executionsTtlSeconds: "1209600"
```

Comment:
Those are not backward compatible. I assume those have the same defaults in the Control Service if not set so the change would not cause issues (crashes) in existing deployments

Reply:
Yes they have the same defaults in the Control Service - https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties